src/custom-types.json

```diff
@@ -473,9 +473,26 @@
   "audio/basic": {
     "compressible": false
   },
+  "audio/gsm": {
+    "extensions": ["gsm"],
+    "sources": [
+      "http://tools.ietf.org/rfc/rfc4856.txt",
+      "http://www.iana.org/assignments/media-types/audio/GSM"
+    ]
+  },
   "audio/l24": {
     "compressible": false
   },
+  "audio/midi": {
+    "extensions":["xmf", "rtttl", "ota", "imy"]
```

Comment:
Please make sure to add a `sources` array here with links to the primary sources for the association of these extensions to this mime type.